config/owasp/suppressions.xml

```diff
@@ -7,5 +7,26 @@
     </notes>
     <cve>CVE-2018-1258</cve>
   </suppress>
- <!-- Supression Group ends here -->
+
+  <suppress until="2021-07-07">
```

Comment:
Looks like an outdated file. Do we need to suppress this?

Reply:
Yes Aravind... these are false positives and dont have fix... they are added as part of OIDC